src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -49,11 +49,10 @@ object EasyIngest {
   private case class Relation(predicate: Predicate, objectSDO: ObjectName = "", `object`: Pid = "", isLiteral: Boolean = false)
   private case class DOConfig(namespace: String, datastreams: List[DatastreamSpec], relations: List[Relation])
 
-  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
-
   def main(args: Array[String]) {
     implicit val s: Settings = Settings(new Conf(args, props))
-    run.get
+    run.doOnError(e => log.error("Ingest failed",e)).
+      doOnSuccess(dict => log.info(s"ingested: ${dict.values.mkString(", ")}"))
```

Comment:
I know I'm kind of a grammer nazi, but put the dot of the previous line on this line 😄